website/docs/docs/develop/dbt-cloud-tips.md

```diff
@@ -0,0 +1,73 @@
+
+**Command bar and status**
+
+You can enter and run commands from the command bar at the bottom of the IDE. Use the [rich model selection syntax](/docs/reference/node-selection/syntax) to [run dbt commands](/docs/reference/dbt-commands) directly within dbt Cloud. You can also view the history, status, and logs of previous runs by clicking **Runs**.
+
+The status icon on the lower right corner of the IDE gives you an indicator of the health of your project. You can identify errors by clicking on the status icon for more details or by clicking **Restart the IDE**.
+
+**Find and replace**
+
+Press Command-F / Control-F to open the find-and-replace bar in the upper right corner of the current file in the IDE. The IDE highlights your search results in the current file and code outline. You can use the up and down arrows to see the match highlighted in the current file when there are multiple matches. To replace the text with something else, use the left arrow.
+
+**DAG in the IDE**
+
+Double-click a node of the directed acyclic graph (DAG) to open that file in a new tab. Click to **Expand** on the upper right side of the DAG and use node selection syntax (`select` or `exclude`) to view a subset of your DAG.
+
+## IDE Keyboard shortcuts
+
+There are default keyboard shortcuts that can help make development more productive and easier for everyone. 
+
+- Command-O / Control-O to select a file to open
+- Press Fn-F1 to view a list of the other editor shortcuts
+- Command-Enter / Control-Enter to Preview
+- Command-Shift-Enter / Control-Shift-Enter to Compile
+
+## Package tips
+
+- Use the [`dbt_codegen`](https://hub.getdbt.com/dbt-labs/codegen/latest/) package to help you generate YML files for your models and sources and SQL files for your staging models.
+- The [`dbt_utils`](https://hub.getdbt.com/dbt-labs/dbt_utils/latest/) package contains macros useful for daily development. For example, `date_spine` generates a table with all dates between the ones provided as parameters.
+- The [`dbt_project_evaluator`](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest) package compares your dbt project against a list of our best practices and provides suggestions and guidelines on how to update your models.
+- The [`dbt_expectations`](https://hub.getdbt.com/calogica/dbt_expectations/latest) package contains many tests beyond those built into dbt Core.
+- The [`dbt_audit_helper`](https://hub.getdbt.com/#:~:text=adwords-,audit_helper,-codegen) package lets you compare the output of 2 queries. Use it when refactoring existing logic to ensure that the new results are identical.
+- The [`dbt_artifacts`](https://hub.getdbt.com/brooklyn-data/dbt_artifacts/latest) package saves information about your dbt runs directly to your data platform so that you can track the performance of models over time.
```

Comment:
```suggestion
- The [dbt_artifacts](https://hub.getdbt.com/brooklyn-data/dbt_artifacts/latest) package saves information about your dbt runs directly to your data platform so that you can track the performance of models over time.
```